pom.xml

```diff
@@ -701,6 +703,12 @@
                 <version>${spring-webflow.version}</version>
             </dependency>
 
+
+            <dependency>
```

Comment:
This dependency is managed by the spring security bom. You don't need to define it.

Reply:
Fixed